deploy/ansible/playbook_04_00_00_db_install.yaml

```diff
@@ -12,6 +12,19 @@
     - vars/ansible-input-api.yaml                               # API Input template with defaults
 
   tasks:
+    - name:                                "refresh BoM Processing: - Create BOM download directories"
```

Comment:
This is centrally handled in https://github.com/Azure/sap-automation/blob/05cd9ae11c456021c7734b161adc68268249dc45/deploy/ansible/roles-sap/3.3.1-bom-utility/tasks/bom-template.yaml#L26